crm_rma_stock_location/__init__.py

```diff
@@ -19,5 +21,14 @@
 #
 ##############################################################################
 
-from . import stock_warehouse
-from . import product
+from . import models
+from . import wizards
+from openerp import SUPERUSER_ID
+
+
+def post_init_hook(cr, registry):
```

Comment:
I see, what you propose is placing it in a separate file.
@osvalr Osval helps us to separate. Just that the file should be named post_init_hook.py, because
init_hook.py sounds like it runs at the beginning of the installation.